test/controllers/api/auth/stories_controller_test.rb

```diff
@@ -1,25 +1,28 @@
 require 'test_helper'
 
 describe Api::Auth::StoriesController do
+  let (:search_term) { 'title' } # factory value
   let (:user) { create(:user) }
   let (:token) { StubToken.new(account.id, ['member'], user.id) }
   let (:account) { user.individual_account }
-  let (:unpublished_story) { account.stories.first }
   let (:published_story) { account.stories.last }
-  let (:latest_story) { create(:story, account: user.individual_account) }
+  let (:latest_story) { create(:story, account: account, description: 'latest') }
   let (:unpublished_story) { account.stories.first }
-  let (:random_story) { create(:story, published_at: nil) }
-  let (:network) { create(:network, account: user.individual_account) }
-  let (:network_story) { create(:story, network_id: network.id, network_only_at: Time.now) }
-  let (:v3_story) { create(:story_v3, account: account) }
+  let (:random_story) { create(:story, published_at: nil, description: 'random') }
+  let (:network) { create(:network, account: account) }
+  let (:network_story) { create(:story, network_id: network.id, network_only_at: Time.now, description: 'network') }
+  let (:v3_story) { create(:story_v3, account: account, description: 'v3_story') }
   let (:released_story) { create(:story, account: account, released_at: Time.now + 1.day) }
 
   before do
     account.stories.each { |s| s }
-    unpublished_story.update!(published_at: nil)
-    published_story.update_attributes!(published_at: 2.days.ago)
-    latest_story.update_attributes!(published_at: 1.day.ago)
-    released_story.update!(published_at: nil)
+    network.stories.each { |s| s.update!(short_description:"network #{s.id}") }
```

Comment:
Can these short descriptions not be set via the factory `create()` calls?  Seems like the other things in here are trying to get around the un-set-ability of `published_at` (which could probably also be addressed in the factory).

Reply:
I was setting the descriptions just to make it easier to debug while testing. I'm happy to remove them now if that's cleaner.

Reply:
Oh, i see, they're not really used in the tests themselves.  Yeah, then remove away.

Reply:
I removed most of the descriptions, but left in a couple just as a way to explicitly get the factory objects indexed in the `begin` block.